cache/cache.go

```diff
@@ -10,8 +10,9 @@ var (
 // Cache represents a generic cache.
 type Cache interface {
 	// Put stores the given (key,value) pair, replacing existing value if
-	// key already exists.
-	Put(key interface{}, value Value) error
+	// key already exists. The return value indicates whether items had to
```

Comment:
Commit body could use some details w.r.t the rationale for this change.

Reply:
Done.